2025-02-17 02:29:45,624 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:30:02,414 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:30:20,159 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:30:40,071 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:31:03,982 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:32:20,878 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:32:52,771 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:33:25,715 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:34:00,637 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:34:39,567 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:35:46,272 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='deployment-ecz5o55dh0dbqagkut47kzyc.stag-vxzy.zettablock.com', port=443): Read timed out. (read timeout=30)")': /main
2025-02-17 02:36:32,175 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:36:48,934 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:37:06,682 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:37:26,432 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:37:50,347 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:27,160 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:43,894 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:01,686 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:21,588 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:45,332 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
